installer-guide/mac-install.md

```diff
@@ -4,38 +4,53 @@
 * Supported version: 0.5.7
 
 While you don't need a fresh install of macOS to use OpenCore, some users prefer having a fresh slate with their boot manager upgrades.
+This guide works for a release of you choice as long as it is listed in the catalog
 
 **Note for legacy users**
 
-* If you want to use OpenCore on a system without UEFI, please follow the [Legacy Install](https://dortania.github.io/OpenCore-Desktop-Guide/extras/legacy.html) section first, after you can continue following the **Base folder structure** section
+* To use OpenCore on an older system with no UEFI (Plain Bios):
+1. First follow the [Legacy Install](https://dortania.github.io/OpenCore-Desktop-Guide/extras/legacy.html) section
+2. Once completed, continue the guide at the **Base folder structure** section
 
-To start we'll want to grab ourselves a copy of macOS, you can skip this and head to formatting the USB if you're just making a bootable OpenCore stick and not an installer. For everyone else, you can either download macOS from the AppStore or with GibMacOS
+* If you are making an OpenCore bootable stick skip this section and head to **Formatting the USB**
 
 ## Downloading macOS
 
-Now lets grab [GibMacOS](https://github.com/corpnewt/gibMacOS) and run the `gibMacOS.command`:
+From a Mac OS machine that meets the requirements of the OS version you want to install, go directly to the Appstore and download the desired OS release the continue and continue to **Setting up the installer**
+
+For machines that cannot download from the Appstore an image or for specific OS releases please use the GibMacOS utility.
```

Comment:
> For machines that cannot download from the Appstore an image or for specific OS releases please use the GibMacOS utility.

I'd prefer this:
`For machines that cannot download from the AppStore or are in need for a specific OS release, you can use the GibMacOS utility.`

Reply:
`For machines that need a specific OS release or can't download from the AppStore, you can use the GibMacOS utility.` might be better @khronokernel 
This sentence is sort of awkward though.

Reply:
Ah yeah that's better